src/py/mat3ra/made/tools/build/deformation/builders.py

```diff
@@ -0,0 +1,63 @@
+from typing import List
+
+from mat3ra.made.material import Material
+from mat3ra.made.tools.build import BaseBuilder
+
+from .configuration import DeformationConfiguration
+
+
+class DeformationBuilder(BaseBuilder):
+    _ConfigurationType: type(DeformationConfiguration) = DeformationConfiguration  # type: ignore
+    _GeneratedItemType: Material = Material
+
+    @staticmethod
+    def deform_slab(configuration):
+        new_material = configuration.slab.clone()
```

Comment:
The logic is ambiguous - why is there slab in configuration for deformation?

Reply:
Changed to SlabDeformationBuilder